src/Tempest/Framework/Config/database.php

```diff
@@ -2,11 +2,39 @@
 
 declare(strict_types=1);
 
+use Tempest\Container\GenericContainer;
 use Tempest\Database\DatabaseConfig;
+use Tempest\Database\DatabaseDialect;
+use Tempest\Database\Drivers\MySqlDriver;
+use Tempest\Database\Drivers\PostgreSqlDriver;
 use Tempest\Database\Drivers\SQLiteDriver;
+use Tempest\Framework\Application\AppConfig;
 
-return new DatabaseConfig(
-    driver: new SQLiteDriver(
-        path: __DIR__ . '/../database.sqlite',
+/** @var AppConfig $app */
+$app = GenericContainer::instance()->get(AppConfig::class);
+
+return match ($app->databaseDialect) {
```

Comment:
I don't think we need to provide this level of configurability: let's keep sqlite as the only option, if people want another driver, they should simply provide their own database config